package.json

```diff
@@ -168,14 +169,14 @@
     "@wdio/spec-reporter": "^5.11.6",
     "awesome-typescript-loader": "^5.2.1",
     "axios-mock-adapter": "^1.16.0",
-    "babel-eslint": "^10.0.2",
+    "babel-eslint": "^10.0.3",
```

Comment:
Can we remove this?  Doesn't appear anything is using it:

> npm list babel-eslint
alchemy-client@0.9.4 C:\Users\dkent\Documents\Projects\GitHub\DAOStack\alchemy
`-- babel-eslint@10.0.2

Reply:
I removed it